modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -227,11 +228,59 @@ - (void)handleSignal:(NSString *)signal
 ////////////////////////////////////////////////////////////////////
 #pragma mark - SendLogs method
 //////////////////////////////////////////////////////////////////// sendLogs
-RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson) {
+RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
+                  jsLogs:(NSString *)jsLogs
+                  callback:(RCTResponseSenderBlock)callback) {
     // TODO: Implement SendLogs for iOS
 #if DEBUG
     NSLog(@"SendLogs() method called, not implemented");
 #endif
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSError *error = nil;
+    NSURL *rootUrl =[[fileManager
+                      URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask]
+                     lastObject];
+
+    NSURL *zipFile = [rootUrl URLByAppendingPathComponent:@"logs.zip"];
+    [fileManager removeItemAtPath:zipFile.path error:nil];
+
+    NSURL *logsFolderName = [rootUrl URLByAppendingPathComponent:@"logs"];
+
+    if (![fileManager fileExistsAtPath:logsFolderName.path])
+        [fileManager createDirectoryAtPath:logsFolderName.path withIntermediateDirectories:YES attributes:nil error:&error];
+
+    NSURL *dbFile = [logsFolderName URLByAppendingPathComponent:@"db.json"];
+    NSURL *jsLogsFile = [logsFolderName URLByAppendingPathComponent:@"Status.log"];
+#if DEBUG
+    NSString *networkDirPath = @"ethereum/mainnet_rpc_dev";
+#else
+    NSString *networkDirPath = @"ethereum/mainnet_rpc";
+#endif
+
+    NSURL *networkDir = [rootUrl URLByAppendingPathComponent:networkDirPath];
+    NSURL *originalGethLogsFile = [networkDir URLByAppendingPathComponent:@"geth.log"];
+    NSURL *gethLogsFile = [logsFolderName URLByAppendingPathComponent:@"geth.log"];
+
+    [dbJson writeToFile:dbFile.path atomically:YES encoding:NSUTF8StringEncoding error:nil];
+    [jsLogs writeToFile:jsLogsFile.path atomically:YES encoding:NSUTF8StringEncoding error:nil];
+
+    //NSString* gethLogs = StatusgoExportNodeLogs();
```

Comment:
Is this commented-out code something we want to keep?

Reply:
yes, will be used soon. It is not added in this PR anyway